src/applications/financial-status-report/tests/unit/transform.unit.spec.jsx

```diff
@@ -0,0 +1,1106 @@
+        {
+          name: 'Pension',
+          amount: '100',
+        },
+        {
+          name: 'Health insurance',
+          amount: '254.45',
+        },
+      ];
+      const allFilters = [
+        'State tax',
+        'Federal tax',
+        'Local tax',
+        '401K',
+        'IRA',
+        'Pension',
+        'FICA (Social Security and Medicare)',
+      ];
+      expect(otherDeductionsAmt(deductions, allFilters)).to.equal(254.45);
+    });
+  });
+
+  // Depends on sumValues, filterDeductions, otherDeductionsAmt
+  describe('getMonthlyIncome helper', () => {
+    it('should return monthy income based on veterans net and other income, and spouses net and other income', () => {
+      expect(getMonthlyIncome(inputObject.data)).to.equal(20398.05);
+    });
+  });
+
+  // depends on sumValues
+  describe('getMonthlyExpenses helper', () => {
+    it('should return a number which is the sum total monthly expenses', () => {
+      const expenses = {
+        expenses: {
+          rentOrMortgage: '100',
+          food: '100',
+        },
+        otherExpenses: [
+          {
+            amount: '100',
+          },
+          {
+            amount: '100',
+          },
+        ],
+        utilityRecords: [
+          {
+            monthlyUtilityAmount: '100',
+          },
+          {
+            monthlyUtilityAmount: '100',
+          },
+        ],
+        installmentContracts: [
+          {
+            amountDueMonthly: '100',
+          },
+          {
+            amountDueMonthly: '100',
+          },
+        ],
+      };
+      expect(getMonthlyExpenses(expenses)).to.equal(800);
+    });
+  });
+
+  // depends on sumValues
+  describe('getEmploymentHistory helper', () => {
+    it('should return a veterans employment history', () => {
+      const history = {
+        questions: {
+          spouseIsEmployed: false,
+          vetIsEmployed: true,
+        },
+        personalData: {
+          employmentHistory: {
+            veteran: {
+              employmentRecords: [
+                {
+                  type: 'Full time',
+                  from: '2017-1-XX',
+                  to: '',
+                  isCurrent: true,
+                  employerName: 'Veteran Current One',
+                },
+              ],
+            },
+          },
+        },
+      };
+
+      const returnVal = getEmploymentHistory(history);
+      expect(returnVal).to.be.an('array');
+      expect(returnVal.length).to.equal(1);
+      expect(returnVal[0].veteranOrSpouse).to.equal('VETERAN');
+      expect(returnVal[0].occupationName).to.equal('Full time');
+      expect(returnVal[0].from).to.equal('01/2017');
+      expect(returnVal[0].to).to.equal('');
+      expect(returnVal[0].present).to.equal(true);
+      expect(returnVal[0].employerName).to.equal('Veteran Current One');
+
+      expect(returnVal[0].employerAddress).to.be.an('object');
+      expect(returnVal[0].employerAddress.addresslineOne).to.equal('');
```

Comment:
Minor nitpick without knowing the context, should these address fields ever be populated with this function?